### DISCUSSION TITLE
Lightburn lbmt files

### DISCUSSION THREAD

**Discussion by Zitt** [shark92651/LaserParamsConverter#16] [Q&A]
I have a file called 80w-120w MOPA Presets.lbmt which I wanted to convert to EZCAD2/3 to use with a my Cloudray 100W mopa. The file appears different than a ?clb? file.

The Cloudray is a EZCad3 only machine... but everything in the community seems to be Lightburn. I was just hoping to get a shortcut to color on stainless steel.

**Comment by shark92651** [comment]
From my understanding, the .lbmt file format is used by the Lightburn Material Test Generator and is not a settings library that LaserParamsConverter would be able to do anything with.
